src/References.js

```diff
@@ -113,9 +115,14 @@ const Footer = styled.div`
     font-family: 'Open Sans Condensed', sans-serif;
     @media ${device.mobileL} { 
         width: 100%;
-        // overflow: hidden;
         position: absolute;
         bottom: 0;
         z-index: 0;
     }
 `;
+const SignOut = styled.div`
+    margin: 3px;
+    color: #fffafa;
+    text-decoration: underline;
```

Comment:
Please add 
`cursor: pointer;`
to the css here so that the cursor appears as a hand when it hovers over this div.